notifications/src/main/java/dk/shape/games/notifications/presentation/viewmodels/notifications/SubjectNotificationSwitcherViewModel.kt

```diff
@@ -11,6 +17,7 @@ import me.tatarka.bindingcollectionadapter2.itembindings.OnItemBindClass
 internal class SubjectNotificationSwitcherViewModel {
```

Comment:
```suggestion
internal class SubjectNotificationSwitcherViewModel(
     private val onItemChangeListener: (item:  Any) -> Unit
) {
```

we can use some listener here instead